inappmessaging/app/src/main/java/com/google/firebase/example/inappmessaging/MyClickListener.java

```diff
@@ -21,3 +21,20 @@ public void messageClicked(InAppMessage inAppMessage, Action action) {
 
 }
 // [END fiam_click_listener]
+
+// [START fiam_click_listener_bundles]
+public class MyClickListener implements FirebaseInAppMessagingClickListener {
```

Comment:
I don't think you can have two classes with the same name (`MyClickListener`) in the same file.  Also should this just be merged into the snippet above anyway?  Just add a line after the one with `metadata` showing how to get the bundle.